sources/niLang/src/Expression.cpp

```diff
@@ -4499,6 +4499,23 @@ tBool DoEvaluate(iExpressionContext*)
 }
 EndOp()
 
+//! FormatTime(TIMEFORMAT: string) the current time.
```

Comment:
Doc is incorrect, it should be `FormatTime(UNIXTIMEINSECS: number, TIMEFORMAT: string)` - please double check that I have the order correct.

Also should add a testcase in Test_Expression.cpp